sdk/cpp/core/src/ydk.cpp

```diff
@@ -117,21 +117,23 @@ static ydk::EncodingFormat get_real_encoding(EncodingFormat encoding)
 {
     switch(encoding)
     {
-        case XML:
-            return ydk::EncodingFormat::XML;
         case JSON:
             return ydk::EncodingFormat::JSON;
+        case XML:
+        default:
```

Comment:
`default` `case`s should be avoided. Suggest removing this or adding cases for all the members of the enum

Reply:
The compiler logged warning message pointing that 'default' case is missing. I was taught always include 'default' case. If not included the 'switch' must be re-factored to if-else statements.